src/components/LineChart.jsx

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {Line} from 'react-chartjs-2';
-import moment from 'moment';
 import {ResizeObserver as ResizeObserverPolyfill} from '@juggle/resize-observer';
+const MONTHS = {
```

Comment:
we still use `moment` in other places, why not keep it here and hardcode month names instead?

Reply:
Moment is only used in one other place and it's relatively trivial to get rid of

Reply:
It's a huge bundle for the front end when it's not even needed. I think I can force graphql to make everything timestamps